packages/xchain-haven/src/haven/api.ts

```diff
@@ -0,0 +1,112 @@
+import axios from 'axios'
+
+// axios.interceptors.request.use((request) => {
+//   // console.log('Starting Request', JSON.stringify(request, null, 2))
+//   console.log(request.url)
+//   return request
+// })
+
+// axios.interceptors.response.use((response) => {
+//   console.log('Response:', JSON.stringify(response.data, null, 2))
+//   //console.log(response.data)
+//   return response
+// })
+
+/**
+ * endpoints for openhaven/backend communication
+ */
+
+export const dataAPI = axios.create()
+
+let API_URL = ''
+let address = ''
+let view_key = ''
+
+export const setAPI_URL = (url: string) => {
+  API_URL = url
+}
+
+export const setCredentials = (address_string: string, viewKey: string) => {
+  address = address_string
+  view_key = viewKey
+}
+
+const CONFIG = {
+  Accept: 'application/json',
+  headers: {
+    'Content-Type': 'application/json',
+  },
+}
+
+export const get_version = (): Promise<any> => {
```

Comment:
`Unexpected any. Specify a different type.eslint`

Pls avoid `any` here and anywhere. Try to define API types (similar to other xchain-*` packages)

Reply:
API Types added